webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -1,6 +1,44 @@
 import Jed from 'jed';
+import { addLocaleData } from 'react-intl';
 import { deprecateObjectProperty } from '../../foreman_tools';
 
+const runningInPhantomJS = () => window._phantom !== undefined;
```

Comment:
note we might be running also in chrome as introduced in 6044a043e1a02f8ff3ba1f51cd2f78faea9d2640, perhaps you can also detect based if ENV["JS_TEST_DRIVER"] is defined.

Reply:
Chrom is not a problem as it implements Intl API correctly. We had issues with phantom that provides Intl object but its functionality is quite limited.